SPECS/gh/gh.spec

```diff
@@ -75,6 +75,9 @@ make test
 %{_datadir}/zsh/site-functions/_gh
 
 %changelog
+* Mon Sep 09 2024 CBL-Mariner Servicing Account <cblmargh@microsoft.com> - 2.13.0-20
+- Bump release to rebuild with go 1.22.7
+
```

Comment:
Seems to be a merge conflict from previous PR:
- https://github.com/microsoft/azurelinux/pull/9969

Reply:
Good point! I'll do a quick cleanup PR for it after this one cherry-picks into main.

Reply:
Reopening this discussion to see if this release number should actually be `-21` since the conflict has two entries of `-19` so one of them will become `-20` on the subsequent cleanup PR fix

Reply:
Good callout, fixed up the changelog and release number for gh.spec